test/031-if.md

```diff
@@ -29,7 +29,7 @@
 
 |Name|Type|Description|Required|
 |----|----|-----------|--------|
-|**postal\_code**||Pattern: ``[A-Z][0-9][A-Z] [0-9][A-Z][0-9]``<br/>||
```

Comment:
Additionally, my changes also change this in the library where the backtick symbol (\`) was being written twice for inline code in markdown (e.g., ``code``), when only a single pair is needed (e.g., `code`).